spec/index.bs

```diff
@@ -3577,6 +3578,10 @@ offset {{GPUOrigin3D}} in texels, used when copying data from or to a {{GPUTextu
     |textureCopyView|.{{GPUTextureCopyView/texture}}.
   - |textureCopyView|.{{GPUTextureCopyView/origin}}.[=Origin3D/x=] must be a multiple of |blockWidth|.
   - |textureCopyView|.{{GPUTextureCopyView/origin}}.[=Origin3D/y=] must be a multiple of |blockHeight|.
+  - If |textureCopyView|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[format]]}}
+    has both depth and stencil aspects,
+    |textureCopyView|.{{GPUTextureCopyView/aspect}} must be
+    {{GPUTextureAspect/"depth-only"}} or {{GPUTextureAspect/"stencil-only"}}.
```

Comment:
"all" should be allowed here for the T2T case
We should only validate the if "depth-only" or "stencil-only" is specified, that the texture actually has a depth or stencil aspect, respectively.

Based on #997 we also need validation that the entire subresource is selected for depth or stencil textures (with OutputAttachment usage TBD..)

We should move the "depth-only" and "stencil-only" part to validation of T2B, B2T and WriteTexture. Probably "all" should be allowed on `depth32float`? So have some language that the `GPUTextureCopyView.aspect` "resolves to" a single aspect.

And put the "all" restriction for T2T copies.  maybe saying that `GPUTextureCopyView.aspect` "resolves to" all the texture aspects